gradlew

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
```

Comment:
Hi @adhes14 , please restore any change to gradlew since this is the wrapper for gradle. Maybe this was accidentaly modified by your editor due an encoding issue.

Reply:
Got it, I missed that part, I see it now. Thanks!

Reply:
I just resolved this issue